src/main/java/org/codehaus/mojo/taglist/TagListReport.java

```diff
@@ -520,17 +528,39 @@ String getExcludesCommaSeparated() {
         }
     }
 
+    void setSourceFileLocale(String sourceFileLocale) {
+        this.sourceFileLocale = sourceFileLocale;
+    }
```

Comment:
You actually don't need the setter unless for testing without DI...

Reply:
it is used in test